.github/workflows/release-v2.yaml

```
@@ -0,0 +1,183 @@
+          #   runner: ubuntu-24.04-arm
+          - goos: darwin
+            goarch: amd64
+            runner: macos-13
+          - goos: darwin
+            goarch: arm64
+            runner: macos-latest
+
+    runs-on: ${{ matrix.runner }}
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)